### PR TITLE
Proper configuration behavior for JsonMessageHandlers for JSON Logging

### LINKED ISSUE #2358: java.lang.NullPointerException com.ibm.ws.http.logging.internal.AccessLogger.log 136
CL Driver: 20180221-1148

bootstrap.properties:
```
com.ibm.ws.logging.message.source=message,trace,accessLog,ffdc
com.ibm.ws.logging.message.format=basic
com.ibm.ws.logging.console.source=message,trace,accessLog,ffdc
com.ibm.ws.logging.console.format=json
```

./server start s1
http://localhost:9080

Two problems observed.
1. A FFDC is generated.

```
Exception = java.lang.NullPointerException
Source = com.ibm.ws.http.logging.internal.AccessLogger.log
probeid = 136
Stack Dump = java.lang.NullPointerException
	at com.ibm.ws.logging.internal.impl.MessageLogHandler.synchronousWrite(MessageLogHandler.java:55)
	at com.ibm.ws.collector.manager.buffer.BufferManagerImpl.add(BufferManagerImpl.java:82)
	at com.ibm.ws.http.logging.source.AccessLogSource$AccessLogHandler.process(AccessLogSource.java:135)
	at com.ibm.ws.http.logging.internal.AccessLogger.log(AccessLogger.java:461)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundServiceContextImpl.logFinalResponse(HttpInboundServiceContextImpl.java:948)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundServiceContextImpl.finishResponseMessage(HttpInboundServiceContextImpl.java:1011)
	at com.ibm.ws.http.channel.internal.outbound.HttpOutputStreamImpl.flushBuffers(HttpOutputStreamImpl.java:492)
	at com.ibm.ws.http.channel.internal.outbound.HttpOutputStreamImpl.close(HttpOutputStreamImpl.java:573)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.tryToCloseStream(HttpDispatcherLink.java:1046)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.closeStreams(HttpDispatcherLink.java:960)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.finish(HttpDispatcherLink.java:940)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.sendResponse(HttpDispatcherLink.java:662)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.displayPage(HttpDispatcherLink.java:508)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.displayLandingPage(HttpDispatcherLink.java:482)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.ready(HttpDispatcherLink.java:358)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.handleDiscrimination(HttpInboundLink.java:530)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.handleNewRequest(HttpInboundLink.java:464)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.processRequest(HttpInboundLink.java:329)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.ready(HttpInboundLink.java:300)
	at com.ibm.ws.tcpchannel.internal.NewConnectionInitialReadCallback.sendToDiscriminators(NewConnectionInitialReadCallback.java:165)
	at com.ibm.ws.tcpchannel.internal.NewConnectionInitialReadCallback.complete(NewConnectionInitialReadCallback.java:74)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager.requestComplete(WorkQueueManager.java:501)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager.attemptIO(WorkQueueManager.java:571)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager.workerRun(WorkQueueManager.java:926)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager$Worker.run(WorkQueueManager.java:1015)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
```

2. No `liberty_ffdc` event was written to console.log

### PR DESCRIPTION
**Pull request by Channyboy**
fixes #2358
#build

### CODE REVIEW

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

File: dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/ConsoleLogHandler.java [hunk 1]
```diff
@@ -36,6 +36,10 @@ public String getHandlerName() {
 
     @Override
     public void synchronousWrite(Object event) {
+        SystemLogHolder sysLogHolder = sysLogHolderOriginal;
+        if (sysLogHolder == null) {
+            return;
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/ConsoleLogHandler.java]: unmatched curly brackets

**Comment by donbourne** [comment]
#build

**Comment by Channyboy** [comment]
Fix delivered through  PR #2362

**Pull request closed without merging**